Tutorial-cli.md

```diff
@@ -377,7 +377,7 @@ The `perms` command provides three operations to list, edit and delete permissio
 
 To list the permissions of your profile resource, we can use the following command:
 ```
-bashlib-auth perms list webid:https://github.com/CommunitySolidServer/CommunitySolidServer
+bashlib-auth perms list webid:
```

Comment:
@mvanbrab an empty prefix seems wrong?

Reply:
The thing after webid: was a copy-paste error (the CSS github repo by the way). Checked this change. The updated command worked. Original did not. Author agreed by the way :-)